ethdb/privateapi/logsfilter.go

```diff
@@ -128,6 +128,7 @@ func (a *LogsFilterAggregator) subscribeLogs(server remote.ETHBACKEND_SubscribeL
 	// Listen to filter updates and modify the filters, until terminated
 	var filterReq *remote.LogsFilterRequest
 	var recvErr error
+	fmt.Printf("filterReq: %+v, recvErr: %+v\n", filterReq, recvErr)
```

Comment:
This needs to be removed before this PR is merged

Reply:
☝️

Reply:
I have removed it, but my push does not seem to reflect on this thread